ansible_runner/utils/streaming.py

```diff
@@ -17,7 +17,11 @@ def stream_dir(source_directory, stream):
                     if relpath == ".":
                         relpath = ""
                     for fname in files:
-                        archive.write(os.path.join(dirpath, fname), arcname=os.path.join(relpath, fname))
+                        try:
+                            archive.write(os.path.join(dirpath, fname), arcname=os.path.join(relpath, fname))
+                        except FileNotFoundError as e:
+                            e.strerror = "invalid symlink detected"
```

Comment:
I think it would be better to raise a new exception with its own message than modifying an existing exception.

That said, perhaps we're going about this wrong.  We should probably just do an `os.path.isfile()` or something like that before doing the `archive.write()`.

Reply:
removed try/except for `if conditional` in latest commit `d9285f6`